server/api.js

```diff
@@ -4,15 +4,140 @@ import pool from "./db";
 
 const router = Router();
 
+// Get all the tasks
 router.get("/tasks", (req, res) => {
 	pool.query("SELECT * FROM tasks").then((result) => {
-		// console.log(result.rows);
 		return res.send(result.rows);
```

Comment:
Before sending back the result, please add a .catch((err)=>{}) block please, incase the server did not send back anything

Reply:
You mean after it.